tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_fillbug.py

```diff
@@ -0,0 +1,27 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+from gaiatest.apps.bzlite.app import BugzillaLite
+
+
+class TestFillBug(GaiaTestCase):
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+        self.test_data = {
+            'name': 'Bugzilla Lite',
+            'url': 'http://bzlite-staging.herokuapp.com/manifest.webapp'}
+        self.logger.info('Test data: %s' % self.test_data)
+        self.marionette.execute_script(
+            'navigator.mozApps.install("%s")' % self.test_data['url'])
+        confirm_install = ConfirmInstall(self.marionette)
+        confirm_install.tap_confirm()
+        self.apps.uninstall('Bugzilla Lite')
```

Comment:
A second note, if we uninstall this app, it will remain uninstall for the rest of the test run. That is to say, if we have a test that is supposed to run against the production environment after that test, we would run the stage version of bzlite.

Hence, I'd suggest we don't remove the bzlite, and we call the app we've just installed 'Bugzilla Lite Stage'. To make the difference clear, here's what I propose:

``` python 
self.test_app = {
    'name': 'Bugzilla Lite Stage',
    'url': 'http://bzlite-staging.herokuapp.com/manifest.webapp'}

def test_fill_new_bug(self):
    bugzilla_lite = BugzillaLiteStage(self.marionette)
    bugzilla_lite.launch()
```

In the page classe:
``` python
class BugzillaLiteStage(BugzillaLite)
    name =  'Bugzilla Lite Stage'
```